bitwarden_license/bit-web/src/app/billing/providers/clients/vnext-manage-clients.component.ts

```diff
@@ -146,7 +146,7 @@ export class vNextManageClientsComponent {
         organization: {
           id: organization.id,
           name: organization.organizationName,
-          seats: organization.seats,
+          seats: organization.seats ? organization.seats : 0,
```

Comment:
You can use `??` here:

```suggestion
          seats: organization.seats ?? 0,
```